F/ForwardDiff/Compat.toml

```diff
@@ -43,7 +43,7 @@ DiffRules = ["0.0.4-0.0.10", "0.1"]
 
 ["0.10.8-0"]
 DiffResults = ["0.0.1-0.0.4", "1.0.1-1"]
-DiffRules = ["0.0.4-0.0.10", "0.1", "1-1.02"]
+DiffRules = ["0.0.4-0.0.10", "0.1", "1-1.0.2"]
```

Comment:
Why not
```suggestion
DiffRules = ["0.0.4-0.0.10", "0.1", "1-1.0"]
```
?

Reply:
I can believe that would work equally well, didn't think of it.